<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>77</width>
      <height>269</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Createbtn">
       <property name="text">
        <string>生成曲线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="linebtn">
       <property name="text">
        <string>隐藏折线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="potbtn">
       <property name="text">
        <string>隐藏插值点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cleanbtn">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="carbtn">
       <property name="text">
        <string>小车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ballbtn">
       <property name="text">
        <string>小球</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>匀速</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>加速</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>减速</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>初速度(5-50)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="vEdit">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>开始运动</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>4</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>插值点：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>tension</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>0.40</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
